client/src/components/ui/itemrow/InviteRow.svelte

```diff
@@ -20,12 +21,18 @@
         email: email,
         office: office,
     };
+
+    const targetName = office ? $allOffices[office] : '';
```

Comment:
Note that if `office`, a `const` variable will not react to it. Here's the fix for that. Moreover, why do we have to check for the truthiness of `office`? Isn't it already non-null to begin with?
```suggestion
    $: targetName = $allOffices[office] ?? '[No Office]';
```

Reply:
No need for reactivity, it is not expected to change since it is a reading from a static snapshot. 
Removing the truthiness check.

Reply:
Although we are not expecting it to change (today!), it is better to accommodate for that use case (for the sake of forward-compatibility) because the `export let` semantics in Svelte are inherently reactive.